pom.xml

```diff
@@ -1,88 +1,144 @@
+  <repositories>
+    <repository>
+      <id>confluent</id>
+      <url>https://packages.confluent.io/maven/</url>
+    </repository>
+  </repositories>
+
+  <organization>
+    <name>Confluent</name>
+    <url>https://www.confluent.io/</url>
+  </organization>
+
+  <properties>
+    <!-- plugins -->
+    <avro-maven-plugin.version>1.9.2</avro-maven-plugin.version>
+    <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+    <!-- libraries -->
+    <kafka.version>2.4.1</kafka.version>
+    <confluent.version>5.4.1</confluent.version>
+    <avro.version>1.9.2</avro.version>
+    <junit.version>4.13</junit.version>
+    <slf4j.version>1.7.30</slf4j.version>
+    <awaitility.version>4.0.2</awaitility.version>
+    <commons-lang3.version>3.10</commons-lang3.version>
+    <testcontainers.version>1.13.0</testcontainers.version>
+    <lombok.version>1.18.12</lombok.version>
+    <javers.version>5.8.12</javers.version>
+    <jackson.version>2.10.3</jackson.version>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka-clients</artifactId>
-      <version>2.2.1</version>
+      <version>${kafka.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-streams</artifactId>
+      <version>${kafka.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>kafka-avro-serializer</artifactId>
+      <version>${confluent.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>${avro.version}</version>
     </dependency>
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
+      <version>${junit.version}</version>
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
-      <version>1.7.26</version>
-      <scope>test</scope>
+      <version>${slf4j.version}</version>
     </dependency>
     <dependency>
       <groupId>org.awaitility</groupId>
       <artifactId>awaitility</artifactId>
-      <version>3.1.6</version>
+      <version>${awaitility.version}</version>
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-      <version>2.6</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.kafka</groupId>
-      <artifactId>kafka-streams</artifactId>
-      <version>2.2.1</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>${commons-lang3.version}</version>
     </dependency>
     <dependency>
       <groupId>org.testcontainers</groupId>
       <artifactId>kafka</artifactId>
-      <version>1.11.3</version>
+      <version>${testcontainers.version}</version>
```

Comment:
you only need to extract the version param for the deps that are referenced in multiple places :)

Reply:
It looks good and all version numbers in one place )